src/system.c

```diff
@@ -573,3 +573,25 @@ int sys_notify_id_valid(int fd, uint64_t id)
 		return -ENOENT;
 	return 0;
 }
+
+/**
+ * Install a file descriptor into the supervisee's process.
+ * @param fd the notification fd
+ * @param addfd the addfd structure
+ *
+ * Install a file descriptor into the supervisee's process. Returns the
+ * installed fd number on success, negative values on failure.
+ *
+ */
+int sys_notify_addfd(int fd, struct seccomp_notif_addfd *addfd)
+{
+	if (state.sup_user_notif <= 0)
+		return -EOPNOTSUPP;
+
+	int rc = ioctl(fd, SECCOMP_IOCTL_NOTIF_ADDFD, addfd);
+	if ( rc < 0 && errno == EINVAL)
+		return -EOPNOTSUPP;
+	if (rc < 0)
+		return -ECANCELED;
+	return rc;
+}
```

Comment:
I'm guessing the little minus-sign-in-a-circle means that this file/patch/PR is missing a newline at the end of the file?  Please check that and add one if it is missing.